src/content/1.7/development/components/grid/tutorials/work-with-bulk-actions.md

```diff
@@ -61,7 +61,7 @@ You can add additional Bulk actions to any Grid using hook. You have to follow t
 
 ```php
 <?php
-// modules/mymodule/mymodule.php
+// modules/my-module/my-module.php
```

Comment:
Is dash allowed in PHP class name?

Reply:
I don't think so, but it's easier to read this way, and I don't think a PrestaShop developer will name its module my-module ... or mymodule :p

Reply:
is it better if I name it "my_module"?

Reply:
https://github.com/PrestaShop/PrestaShop/blob/4c4f83de191fa30b364e6242da5005b4503a780c/classes/Validate.php#L236

Reply:
> is it better if I name it "my_module"?

Yeah, if someone wants to play around and create testing module, it's very likely they will copy names from docs. :)

Reply:
Agree with @Matt75 I hate underscore, more since it's part of `\w` (in regex context) which means `my_module` is one word, and `my-module` are two words.
Directory `my-module` => class name `MyModule`

Reply:
> Directory my-module => class name MyModule

For PS modules you cannot have directories with dash as they must match module class name.

Reply:
🤢  We maybe can do something to improve this behavior no?

Reply:
Probably, but not in these pull request as "docs" are not in the same repository of "core classes" :'(

Reply:
I think you can you either `my_module` or `mymodule` or even `demo` or `example`. The main point is to have valid module name, so docs are not misleading.

Reply:
@mickaelandrieu I'll have a try today to use dash instead of underscore!

Reply:
We can't use dash because of this function https://github.com/PrestaShop/PrestaShop/blob/develop/classes/module/Module.php#L1142 
But still weird since we accept the dash in the module name -_-' 
Let's improve this for the next release!

Reply:
solved